mobile/squads/squad_members.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { useState, useEffect } from "react";
-import { View, Text, FlatList} from "react-native";
+import { View, Text, FlatList, Image} from "react-native";
 import { squad_members_styles } from "./squad_members_styles";
 import {callBackend } from "../backend/backend"
 
 type Users = {
-    name: string
+    name: string,
+    photo: string
```

Comment:
Lets just move these fields into the one `User` type

Reply:
done